packages/components/src/index.ts

```diff
@@ -20,7 +20,7 @@ export * from "./ErrorPage"
 export * from "./FieldGroup"
 export * from "./FieldMessage"
 export * from "./Filter"
-export * from "./GuidanceBlock"
```

Comment:
Do we want to leave this here for to allow folks to transition to the versioned structure?

Reply:
nvm, missed the bottom import